packages/edit-site/src/components/site-hub/style.scss

```diff
@@ -36,8 +36,17 @@
 
 	&:hover,
 	&:focus,
-	&:active {
-		color: $gray-200;
```

Comment:
I think this color needs to be kept at least for the :active state otherwise when clicking the link, the text will become black on a black background. Screenshot:
![Screenshot 2024-06-24 at 11 34 43](https://github.com/WordPress/gutenberg/assets/1682452/2a8c27ca-cda6-458d-8372-0583d967ee94)

Reply:
Also, the button padding should be adjusted to keep the current spacing as much as possible. Screenshot with comparison of current trunk and this PR:

![Screenshot 2024-06-24 at 11 34 55](https://github.com/WordPress/gutenberg/assets/1682452/258311e6-e892-4673-befe-f41f698dc556)